functions/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "functions",
+  "description": "Cloud Functions for Firebase",
+  "scripts": {
+    "lint": "eslint .",
+    "serve": "firebase emulators:start --only functions",
+    "shell": "firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "12"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "@brown-ccv/disco-styles": "^2.2.2",
+    "@brown-ccv/disco-vue-components": "1.4.3",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/pro-duotone-svg-icons": "^5.15.1",
```

Comment:
do we definitely need all the font awesome here?

Reply:
I first added the core, then it complaint about the free-solid, so then added the rest, so potentially it doesn't need the free-brands or pro-duotone, but I think it needs whatever the website uses? The deploy is a bit slow, so I got lazy to try one by one

Reply:
maybe we open an issue for the someday list?

Reply:
Definitely, we can refine the list when we integrate the components and also get rid of fontawesome pro